std/algorithm/comparison.d

```diff
@@ -887,12 +887,23 @@ template equal(alias pred = "a == b")
 
     enum hasFixedLength(T) = hasLength!T || isNarrowString!T;
 
+    // use code points when comparing two ranges of UTF code units that aren't
+    // the same type. This is for backwards compatibility with autodecode
+    // strings.
+    enum useCodePoint(R1, R2) =
```

Comment:
`private`

Reply:
It's inside an  eponymous template. Can't access it.